tests/bootstrap.php

```diff
@@ -7,6 +7,10 @@
     exit;
 }
 
+if (!defined('APP_UNDER_TEST')) {
+    define('APP_UNDER_TEST', true);
+}
+
```

Comment:
Hey guys I don't have my windows environment set up to run these test so I'm guessing at a fix for the registration modal blocking test menu clicks.
If my last attempt doesn't work, you guys may need to jump in and help me out. thanks!